packages/vitest/src/runtime/workers/base.ts

```diff
@@ -53,6 +95,7 @@ export async function setupEnvironment(context: WorkerSetupContext): Promise<()
   }
   const otel = context.traces
 
+  // TODO: if viteModuleRunner is disabled, disable this
```

Comment:
Should this disable environment or use `NativeModuleRunner` for environment loader as per docs?